Unity/opentok/OTXCFramework.podspec

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+  s.name         = "OTXCFramework"
+  s.version      = "2.27.0-custom.0"
+  s.summary      = "OpenTok lets you weave interactive live WebRTC video streaming right into your application"
+  s.description  = <<-DESC
+                   The OpenTok iOS SDK lets you use WebRTC video sessions in apps you build for iPad,
+                   iPhone, and iPod touch devices.
+                   DESC
+  s.homepage     = "https://tokbox.com/developer/sdks/ios/"
+  s.license      = { :type => "Commercial", :text => "https://tokbox.com/support/tos" }
+  s.author       = { "TokBox, Inc." => "support@tokbox.com" }
+
+  s.platform     = :ios
+  s.ios.deployment_target = '13.0'
+  s.source       = { :http => 'file:' + __dir__ + '/OpenTok-4af7028dc.xcframework.zip' }
```

Comment:
lets remove the random hash... so we wont need to change this file.

Reply:
It is not random. It is the commit in the branch that the binary was created from (https://github.com/opentok/native-sdk-src/tree/CMPF-40).